src/Plugins/Parallel/Handlers/Laravel.php

```diff
@@ -43,10 +46,8 @@ private static function whenUsingLaravel(array $arguments, Closure $closure): ar
     {
         $isLaravelApplication = InstalledVersions::isInstalled('laravel/framework', false);
         $isLaravelPackage = class_exists(\Orchestra\Testbench\TestCase::class);
-        if ($isLaravelApplication) {
-            return $closure($arguments);
-        }
-        if ($isLaravelPackage) {
+
+        if ($isLaravelApplication && ! $isLaravelPackage) {
```

Comment:
@nunomaduro noticed you changed the logic here, but AFAIK we don't want to run the Laravel test runner in a package context because it won't work.

Reply:
Ups! Good point, can you update this?

Reply:
Already have :-) It won't apply the Laravel runner anymore if it is a Laravel package.